doc/tools/create_changelog.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/python
+from __future__ import absolute_import
+from __future__ import unicode_literals
+from __future__ import print_function
+import sys
+import io
+from subprocess import check_output, CalledProcessError
```

Comment:
Yeah, I guess I could go even farther and explicitly use the ```from a import b``` syntax on _all_ the imports.  If I make a long pass on this stuff again I'll clean that up.